privacy/views/data_protection_menu_view.xml

```diff
@@ -0,0 +1,60 @@
+<!-- Copyright 2018 Eficent Business and IT Consulting Services S.L.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo>
+    <record id="action_data_protection_partner_form" model="ir.actions.act_window">
+        <field name="name">Partners</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">res.partner</field>
+        <field name="view_mode">kanban,tree,form</field>
+    </record>
+    <menuitem
+        id="parent_menu_data_protection"
+        name="Privacy"
+        groups="group_data_protection_user"
+        web_icon="privacy,static/description/icon.png"
+    />
+    <menuitem
+        id="menu_data_protection_master_data"
+        name="Master Data"
+        parent="parent_menu_data_protection"
+        groups="group_data_protection_user"
+        sequence="10"
+    />
+    <menuitem
+        id="menu_data_protection_partner"
+        name="Partners"
+        parent="menu_data_protection_master_data"
+        action="action_data_protection_partner_form"
+        sequence="1010"
+    />
+    <menuitem
+        id="menu_data_protection_transaction"
+        name="Transactions"
+        parent="parent_menu_data_protection"
+        groups="group_data_protection_user"
+        sequence="20"
+    />
+    <menuitem
+        id="menu_data_protection_report"
+        name="Reports"
+        parent="parent_menu_data_protection"
+        groups="group_data_protection_user"
+        sequence="30"
+    />
+    <record id="action_data_protection_config" model="ir.actions.act_window">
+        <field name="name">Data Protection Settings</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">res.config.settings</field>
+        <field name="view_mode">form</field>
+        <field name="target">inline</field>
+        <field name="context">{'module': 'data_protection'}</field>
+    </record>
```

Comment:
This menu item leads to settings, yes, but there are no privacy-related settings. Is that normal? Maybe it's a submodule that needs this hook here? Otherwise it could be removed.